gcc/rust/resolve/rust-ast-resolve-pattern.h

```diff
@@ -70,11 +70,9 @@ class PatternDeclaration : public ResolverBase
       pattern.get_node_id (), pattern.get_locus (), type);
   }
 
-  void visit (AST::WildcardPattern &pattern) override
+  void visit (AST::GroupedPattern &pattern) override
```

Comment:
Its looks like the wildcard pattern visitor was removed maybe you just need to revert that change and just add the new grouped pattern visitor which is a nice single commit